### DISCUSSION TITLE
[Feature] Utilitiy for incorrect faces based on birthday

### DISCUSSION THREAD

**Discussion by EnochPrime** [immich-app/immich#13065] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Create a new utility that looks for persons who are tagged in images taken prior to their date of birth. Raise these to be corrected by tagging the face with the proper person.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
These days the algorithm will not tag any photos before someone's birthdate, so a utility like this would only apply to tags from before that was implemented.
